managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerTest.java

```diff
@@ -3624,6 +3624,8 @@ public void testDoNotGetOffloadPoliciesMultipleTimesWhenTrimLedgers() throws Exc
         LedgerOffloader ledgerOffloader = mock(NullLedgerOffloader.class);
         OffloadPoliciesImpl offloadPolicies = mock(OffloadPoliciesImpl.class);
         when(ledgerOffloader.getOffloadPolicies()).thenReturn(offloadPolicies);
+        when(ledgerOffloader.offload(any(), any(), any())).thenReturn(CompletableFuture.completedFuture(null));
+        when(ledgerOffloader.getOffloadPolicies().getManagedLedgerOffloadThresholdInBytes()).thenReturn(-1L);
```

Comment:
```suggestion
        when(offloadPolicies.getManagedLedgerOffloadThresholdInBytes()).thenReturn(-1L);
```
may only add this logic the test can become stable.
leger.close() will invoke `ledgerOffloader.getOffloadPolicies().getManagedLedgerOffloadThresholdInBytes()`

the first ledgerOffloader will open a timer task to call
https://github.com/apache/pulsar/blob/fa328a42d5770a27237d926eac97385284876174/managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java#L2395
so we don't open this timer task is enough

Reply:
If not add 
```
when(ledgerOffloader.offload(any(), any(), any())).thenReturn(CompletableFuture.completedFuture(null))
```
The test will throw NPE

Reply:
@congbobo184

Reply:
I see the test don't sync invoke method offload, so it don't influence the test result.